CompetitionTemplate/src/main/java/competition/subsystems/elevator/commands/MoveToMaxHeightCommand.java

```diff
@@ -0,0 +1,50 @@
+package competition.subsystems.elevator.commands;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import competition.subsystems.elevator.ElevatorSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.math.PIDFactory;
+import xbot.common.math.PIDManager;
+
+@Singleton
+public class MoveToMaxHeightCommand extends BaseCommand {
+	
+	ElevatorSubsystem elevator;
+	PIDManager pid;
+	
+	
+	double max = 90;
```

Comment:
We should ask the elevatorSubsystem what the max is - the command probably shouldn't know.